work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -1917,7 +1917,136 @@ def tree_reduce(self, fn, seq, chunk_size=2):
             seq = results
 
         return seq[0]
-           
+
+    # map, but remote
+    def Rmap(self, fn, array, name, chunk_size=1):
```

Comment:
Let's rename all of these to `remote_` (e.g. `remote_map`), as an initial capital letter is meant for classes, no functions.